ThreeBotPackages/zerobot/admin/actors/notifications.py

```diff
@@ -8,4 +12,7 @@ def _init(self, **kwargs):
 
     @j.baseclasses.actor_method
     def check_new_release(self, schema_out=None, user_session=None):
-        return j.data.serializers.json.dumps(Tools.get_newest_version())
+        if Tools.is_latest_release():
```

Comment:
also could be replaced with `j.core.tools.is_latest_release()`

Reply:
thanks Waleed :+1: